app/helpers/remote_execution_helper.rb

```diff
@@ -96,7 +96,7 @@ def remote_execution_provider_for(template_invocation)
   def job_invocations_buttons
     [
       documentation_button_rex('3.2ExecutingaJob'),
-      display_link_if_authorized(_('Run Job'), hash_for_new_job_invocation_path)
+      new_link(_('Run Job'), hash_for_new_job_invocation_path)
```

Comment:
no need to pass the hash here, it get automatically calculated according to the controller